buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle

```diff
@@ -43,7 +44,7 @@ tasks.named('test') {
     useJUnitPlatform {
         excludeTags "RemoteTest", "integrationTest", "KarateTest"
     }
-    failFast = true
+    failFast = false
```

Comment:
The false setting likely gives us a clearer picture in CI reports

Reply:
Fjernet